tests/test_selective_nodejs_provider.py

```diff
@@ -0,0 +1,25 @@
+import os
+import subprocess
+
+from utils import constants
+from utils.command import build_analysis_command
+from utils.report import get_json_from_report_output_file
+
+
+# Polarion TC MTA-542
+def test_nodejs_provider_analysis_with_app(nodejs_analysis_data):
```

Comment:
Can you open a different PR to rename this method and the `go` to remove the `with_app`? it is a little redundant.

`test_nodejs_provider_analysis`